TrackPropagation/RungeKutta/src/CartesianStateAdaptor.h

```diff
@@ -1,6 +1,9 @@
 #ifndef CartesianStateAdaptor_H
 #define CartesianStateAdaptor_H
 
+#include "RKSmallVector.h"
```

Comment:
Better to include headers with the full path
(I see that this is already not always satisfied by the current code: @mrodozov do you want to propagate it, at least into the files touched by this pull request?)

Reply:
@perrotta I agree in general (i.e. for headers in `interface`), but why would it be better for headers in `src` (or `plugins`) that ought to be implementation details of the library (plugin) and not to be `#include`'d elsewhere?